sample/src/main/java/co/nimblehq/sample/AuthService.kt

```diff
@@ -1,13 +1,12 @@
 package co.nimblehq.sample
 
-import co.nimblehq.sample.entity.UserEntity
+import co.nimblehq.sample.entity.TokenEntity
 import co.nimblehq.survey.sdk.request.LoginRequest
 import retrofit2.http.Body
 import retrofit2.http.POST
 
 interface AuthService {
 
     @POST("/api/v1/oauth/token")
-    suspend fun loginEmail(@Body request: LoginRequest): UserEntity
-
+    suspend fun loginEmail(@Body request: LoginRequest): TokenEntity
```

Comment:
It completely breaks the Sdk structure, the sdk must not depend on any resource outside the scope.

Reply:
@hoangnguyen92dn Sorry, I didn't not get your point. `AuthService` and `TokenEntity` are both from `sample-app`. The Sdk does not depend on them.

Reply:
@doannimble 🤦 Ah sorry. My bad. So it's ok now